cypress/integration/SignIn/signIn.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { formSelectors } from './selectors'
+
+describe('Sign In Page', () => {
+  context('given valid user credential', () => {
+    beforeEach(() => {
+      cy.mockSignInResponse(200, 'signInResponse/valid_response.json')
+      cy.signIn(Cypress.env('CYPRESS_VALID_EMAIL'), Cypress.env('CYPRESS_VALID_PASSWORD'))
+    })
+
+    it('stores token to local storage', () => {
+      cy.findByTestId(formSelectors.button).click().should(() => {
+        expect(localStorage.getItem('access_token')).eq('test-access-token')
+        expect(localStorage.getItem('refresh_token')).eq('test-refresh-token')
+        expect(localStorage.getItem('token_type')).eq('Bearer')
+      })
+    })
+
+    it('redirects to home page', () => {
+      cy.findByTestId(formSelectors.button).click()
+
+      cy.url().should('include', '/')
+    })
+  })
+
+  context('given invalid user credential', () => {
+    before(() => {
+      cy.mockSignInResponse(400, 'signInResponse/error_response.json')
+      cy.signIn('invalid@email.com', 'invalidPassword')
+      cy.findByTestId(formSelectors.button).click()
```

Comment:
Same as the above comment 😄

Reply:
I think we can keep this one as it is `before()` hook. UI test setup is expensive so for some test cases I think it is ok to use `before` hook.